app/controllers/BoxNumberController.scala

```diff
@@ -82,4 +91,14 @@ class BoxNumberController @Inject()(identify: IdentifierAction,
     }
   }
 
+  private def createRadioButton(form: Form[_], values: Seq[String])(implicit messages: Messages): Seq[RadioItem] = {
```

Comment:
Rename the `values` parameter to represent what type of data it holds e.g. `boxNumbers`